hack/run-operator-tests.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+INTEGRATION_OUTPUT_JUNIT=${INTEGRATION_OUTPUT_JUNIT:-false}
+KUDO_VERSION=${KUDO_VERSION:-test}
```

Comment:
This is to allow users to override `KUDO_VERSION`. The default value of `test` is used as a tag for the Docker image that is build and injected to the `kind` cluster. This is to avoid clashes with existing Docker images as `test` is only used in this context.